requirements_dev.txt

```diff
@@ -2,7 +2,8 @@
 # code linting and formatting
 flake8
 flake8-docstrings
-flake8-black
+# Black 22.1 has a fatal bug when used with flake8-black, so skip it for now.
+#flake8-black
```

Comment:
This works better (it's about the one combination you DIDN'T try!):

```suggestion
# Black 22.1 has a fatal bug when used with flake8-black, so freeze it for now.
black<=21.12b0
flake8-black
```

I've also pulled this into a separate PR (#2513) that we should merge first.